ambry-api/src/test/java/com.github.ambry/rest/RestUtilsTest.java

```diff
@@ -650,19 +723,40 @@ private String generateRandomString(int length) {
    * Any other headers have to be set explicitly.
    * @param headers the {@link JSONObject} where the headers should be set.
    * @param ttlInSecs sets the {@link RestUtils.Headers#TTL} header.
-   * @param isPrivate sets the {@link RestUtils.Headers#PRIVATE} header. Allowed values: true, false.
    * @param serviceId sets the {@link RestUtils.Headers#SERVICE_ID} header.
    * @param contentType sets the {@link RestUtils.Headers#AMBRY_CONTENT_TYPE} header.
    * @param ownerId sets the {@link RestUtils.Headers#OWNER_ID} header. Optional - if not required, send null.
+   * @param insertAccount {@code true} if {@link Account} info has to be injected into the headers.
+   * @param insertContainer {@code true} if {@link Container} info has to be injected into the headers.
    * @throws JSONException
    */
-  private void setAmbryHeadersForPut(JSONObject headers, String ttlInSecs, String isPrivate, String serviceId,
-      String contentType, String ownerId) throws JSONException {
+  private void setAmbryHeadersForPut(JSONObject headers, String ttlInSecs, String serviceId, Container container,
```

Comment:
Missing the @param for `container`. Btw why container needs to be supplied but account is the UNKNOWN one?

Reply:
This just keeps the old behavior wrt account. The container is being explicitly sent here because its private property (apart from only the id which used to matter before) is now important. However with account, one account is no different from the other.